fabric/dataclient.go

```diff
@@ -387,104 +387,22 @@ func convertOne(fg *Fabric) ([]*eskip.Route, error) {
 			lbEndpoints: endpoints,
 		}
 
-		defaultPrivileges := []interface{}{
+		defaultScopePrivileges := []interface{}{
 			"uid",
 		}
-		for _, priv := range fg.Spec.AllowList {
-			defaultPrivileges = append(defaultPrivileges, priv)
+		var defaultAllowList []interface{}
+		for _, app := range fg.Spec.AllowList {
+			defaultAllowList = append(defaultAllowList, "sub", app)
```

Comment:
scope and allowList needs to be split because we have to use 2 different filters